lib/util/env.ex

```diff
@@ -141,6 +141,31 @@ defmodule Antikythera.Env do
     end
   end
 
+  if Antikythera.Env.Mapping.cloud?(@compile_env) do
+    @scheme "https"
```

Comment:
Optional:
It looks like Antikythera isn't support `http` on cloud for both of Antikythera's domain and a custom domain of gears.
It would be better if `default_base_url/2` uses this to share the above knowledge.

Reply:
`default_base_url/2` takes `env` as the second argument. Do you mean that the following code is better?

```elixir
@doc """
...
Whether the scheme of the URL is `https` or `http` depends on whether `Antikythera.Env.Mapping.cloud?/1` returns true or false.
"""
def base_url(conn)

@doc """
...
Whether the scheme of the URL is `https` or `http` depends on whether `Antikythera.Env.Mapping.cloud?/1` returns true or false.
"""
def default_base_url(gear_name), do: default_base_url(gear_name, @compile_env)

@doc false
def default_base_url(gear_name, env)
```

Reply:
I suppose use cases of `default_base_url/1` and `default_base_url/2` are different.
The former is used in the running gear as same as `base_url/1`, and the latter may be used from outside the gear (connecting to the deployed gear from local for example, so we need to indicate `env` explicitly).
Therefore I think separating interface like above example would be good.

Reply:
Sorry, I missed your comment.

I meant that we should use `@scheme` in default_base_url as well:
```elixir
  defun default_base_url(
          gear_name :: v[GearName.t() | GearNameStr.t()],
          env :: v[t] \\ @compile_env
        ) :: Url.t() do
    if Mapping.cloud?(env) do
      @scheme <> "://" <> Routing.default_domain(gear_name, env)
    else
      @scheme <> "://" <> Routing.default_domain(gear_name, env) <> ":#{port_to_listen()}"
    end
  end
```

Since we always add `"://"` to `@scheme`, you can change `@scheme` to `"https://"` and its name to `@base_url_prefx` or something.

Reply:
One gear uses `default_base_url(gear_name, env)` to get the deployment environment URL from local mix task.
So it is necessary to decide the scheme based on the `env` argument.

Reply:
Ah, sorry. I misunderstood `default_base_url/2`.
You're right!